### PR TITLE
adds build for nodejs 8

### PR DESCRIPTION
**Pull request by sparty02**

### CODE REVIEW

File: 8/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+      io.k8s.display-name="Node.js $NODEJS_VERSION" \
+      io.openshift.expose-services="8080:http" \
+      io.openshift.tags="builder,nodejs,nodejs$NODEJS_VERSION" \
+      com.redhat.dev-mode="DEV_MODE:false" \
+      com.redhat.deployments-dir="/opt/app-root/src" \
+      com.redhat.dev-mode.port="DEBUG_PORT:5858"\
+      com.redhat.component="rh-nodejs8-docker" \
+      name="centos/nodejs-8-centos7" \
+      version="8" \
+      release="1" \
```
Comment (sparty02) [inline 8/Dockerfile]: @hhorak  Not totally sure what this release signifies

File: 8/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+      maintainer="SoftwareCollections.org <sclorg@redhat.com>"
+
+RUN yum install -y centos-release-scl-rh && \
+    yum-config-manager --enable centos-sclo-rh-testing && \
+    INSTALL_PKGS="rh-nodejs8 rh-nodejs8-npm rh-nodejs8-nodejs-nodemon nss_wrapper" && \
```
Comment (sparty02) [inline 8/Dockerfile]: This assumes that the nodejs8 packages have been pushed to the rpm repos

File: 8/Dockerfile.rhel7 [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+      io.k8s.display-name="Node.js $NODEJS_VERSION" \
+      io.openshift.expose-services="8080:http" \
+      io.openshift.tags="builder,nodejs,nodejs$NODEJS_VERSION" \
+      com.redhat.dev-mode="DEV_MODE:false" \
+      com.redhat.deployments-dir="/opt/app-root/src" \
+      com.redhat.dev-mode.port="DEBUG_PORT:5858" \
+      com.redhat.component="rh-nodejs8-docker" \
+      name="rhscl/nodejs-8-rhel7" \
+      version="8" \
+      release="14.1"
```
Comment (sparty02) [inline 8/Dockerfile.rhel7]: @hhorak  Not totally sure what this release signifies

File: 8/Dockerfile.rhel7 [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+
+# To use subscription inside container yum command has to be run first (before yum-config-manager)
+# https://access.redhat.com/solutions/1443553
+RUN yum repolist > /dev/null && \
+    yum-config-manager --enable rhel-server-rhscl-7-rpms && \
+    yum-config-manager --enable rhel-7-server-optional-rpms && \
+    yum-config-manager --enable rhel-7-server-ose-3.2-rpms && \
+    yum-config-manager --disable epel >/dev/null || : && \
+    INSTALL_PKGS="rh-nodejs8 rh-nodejs8-npm rh-nodejs8-nodejs-nodemon nss_wrapper" && \
```
Comment (sparty02) [inline 8/Dockerfile.rhel7]: This assumes that these packages have been pushed to the yum repos

**Comment by sparty02** [comment]
@hhorak I submitted this PR because I wanted to get this on the radar since NodeJS 8 will be the LTS version soon.  I used v6 as a reference and just changed the relevant parts for v8, but I left some comments in the PR to get some feedback, primarily confirming that the yum packages for nodejs 8 will be available

**Comment by hhorak** [comment]
Thank you. Please, don't take this as any promise, but we're working on node 8, we should have at least centos packages available soon. As for the feature-set, it should be very similar to v6 container image.

Anyway, we'll probably do the PR a bit differently, since adding new files would remove the history from the files. So, please, don't be offended when we create a similar, but slightly different PR..

**Comment by sparty02** [comment]
No problem!  Thanks for the reply

**Comment by hhorak** [comment]
There is now a new PR #156 that should keep history in 8/ directory, so closing this one.

**Pull request closed without merging**